deeprank2/query.py

```diff
@@ -265,12 +265,11 @@ def _build_helper(self) -> Graph:
         structure = self._load_structure()
 
         # find the variant residue and its surroundings
-        variant_residue: Residue = None
         for residue in structure.get_chain(self.variant_chain_id).residues:
             if residue.number == self.variant_residue_number and residue.insertion_code == self.insertion_code:
                 variant_residue = residue
                 break
-        if variant_residue is None:
+        else:  # if break is not reached
```

Comment:
no, this is correct. 
The `else` is activated if the for-loop doesn't break (i.e. not a in case the if-statement is not evaluated). It's kind of an [uncommon use of `else`](https://python-notes.curiousefficiency.org/en/latest/python_concepts/break_else.html) and confusing that the same keyword is used, but it can be pretty useful (like here).

Reply:
oh interesting, never seen it!